packages/docs-data/compile-docs-data.js

```diff
@@ -80,8 +80,7 @@ function transformDocumentalistData(key, value) {
         // reverse the list so highest version is first (easier indexing)
         return Array.from(majors.values()).reverse();
     }
-    // remove all class declarations as they're currently unused and only increase bundle size
-    if (value != null && value.kind !== "class") {
```

Comment:
I've had to change this to include the class members but there might be a more efficient way to do it?

Reply:
I think this is fine